### ISSUE TITLE
Improve Styling of Website

### ISSUE THREAD

**Issue by kartikvirendrar** [kartikvirendrar/stocksearch#1]

**Comment by nIMblEt06** [comment]
hey, in what ways would you want me to improve styling of your website? 
also, could you please create seperate issues for every new feature that you might require so that i can contribute in a more organised manner? 
thanks!

**Comment by kartikvirendrar** [comment]
@nIMblEt06 Yes, I will create separate issues for the same

**Comment by nIMblEt06** [comment]
@kartikvirendrar have designed new UI for the website, please have a look at the same and accept a non-code PR from my end.
https://www.figma.com/file/r1JIx3AcKJlNj9eP1N8Qff/hacktoberFestContribution?node-id=0%3A1

**Comment by kartikvirendrar** [comment]
@nIMblEt06 fork this repository and create a new folder named FIGMA and add your figma design link to it in YOUR_NAME.md, then create a PR

**Comment by atharva-gadekar** [comment]
Hello @kartikvirendrar can you please assign this issue to me? I have made quite a few websites. One example would be https://atharva-gadekar.github.io/WebD-Selection/code/home.html
I would love to make this a better-looking website. This is my first open source contribution, it will help me learn a lot. Thankyou

**Comment by atharva-gadekar** [comment]
I could also try to convert this project to Nextjs or Reactjs although there is a tiny possibility that I might need a little explanation about the API workings. So, if you want, you can assign it to me. I will try my best to work on it. Thankyou

**Comment by kartikvirendrar** [comment]
Here I am using API's to fetch the real time stock data from internet, You can use fetch or Axios to fetch API in ReactJs or NextJS

**Comment by atharva-gadekar** [comment]
Hello, @kartikvirendrar this is the current designing for the website from my side. I am yet to fetch the stock details from the API, but nevertheless, I wanted to share this design with you. Please give me some suggestions over the same.
![image](https://user-images.githubusercontent.com/97454406/193399485-551372d3-9a8f-402d-9404-12182174d610.png)

**Comment by atharva-gadekar** [comment]
I will try to create a pull request by the end of today after I complete the API fetching part. I also wanted to redesign the Nav bar as the current one won't go with this theme. The footer, however, would be fine. Please suggest if I should over ride and redesign the current navbar and please give me your suggestions over the current design.

**Comment by atharva-gadekar** [comment]
I also wanted to ask, if can I remove the JSON.stringify(stockls[${x}]) part as it looks really weird as of now. In future commits, I might try to open one stock on an entire page and display all the other details, but as of now, I wanted to remove that feature.

**Comment by kartikvirendrar** [comment]
yes you can remove the stringify part, design can be improved a little

**Comment by atharva-gadekar** [comment]
Hello @kartikvirendrar I have merged my design changes with the recent commits. Please merge my pull request. I will style the footer after you assign me the "Add a footer" issue. Hoping that you like the current changes
![image](https://user-images.githubusercontent.com/97454406/193411925-84ed955f-5114-4164-a90e-4cb7cb9257c4.png)
